### DISCUSSION TITLE
Chat doesn't work when attaching a file.

### DISCUSSION THREAD

**Discussion by breissie** [Genuitec/Copilot4Eclipse-community#45] [Help] [answered]
I am attempting to ask a question via the chat panel  and attaching a file for reference. However, when that happens, I get the following error:

[2024-11-01T20:10:31.933862300-04:00] LANGUAGE_SERVER_TO_LSP4E com.genuitec.copilot4eclipse.copilot:
{"jsonrpc":"2.0","id":"73","error":{"code":-32602,"message":"Schema validation failed with the following errors:\n- /references/0: Expected union value"}}

This is with version 1.3.1

**Comment by wayneparrott** [comment, marked as answer]
Thanks for sharing this issue. I am able to replicate it and have identified a small protocol change. A fix will be available in the next update.

Wayne
Copilot4Eclipse Team
